lib/rubygems/basic_specification.rb

```diff
@@ -256,6 +256,14 @@ def platform
     raise NotImplementedError
   end
 
+  def installable_on_platform?(local_platform)
+    return true if [Gem::Platform::RUBY, nil].include?(platform)
```

Comment:
You used an array inclusion here, two equalities with `||` in the Bundler method, and the current implementation uses two separate early returns. Unify it?

Reply:
Also, maybe `target_platform` rather than `local_platform` since I don't think this is always the local platform (and if it was, we wouldn't need a parameter since that's constant).